### DISCUSSION TITLE
[Question]: How can I only get snapshots of motion saved.

### DISCUSSION THREAD

**Discussion by flyingwalrushippo** [blakeblackshear/frigate#13151] [Ask A Question]
### What is your question:

Prior to this new update, i did not save motion recordings. All i saved was image snapshots on frigate whenever motion was detected and it was all logged under the events tab. How can i restore this functionality. I don't need videos simply because I have another system for 247 video recording. I just want frigate to take pictures when actual motion is being detected and timestamp it for me.

**Comment by hawkeye217** [comment]
See the [FAQ](https://github.com/blakeblackshear/frigate/discussions/12724), Usage question 8.

**Reply by dopeytree** [reply]
I think it's nonsense. Frigate could easily separate out the images like it does for frigate+ and you just review them all. If you press a button it links to the video (if available).

**Reply by NickM-27** [reply]
A preview that shows what happened in a sped up form provides much more information than a couple images. This way it's also clear that these happened at the same time so it's easier and faster to review periods of time

**Reply by dopeytree** [reply]
Can't print a video etc... 

but anyway have worked out we can just right click images in frigate+ section and save them so a solution has been acquired.🤓
